src/cmd/singularity/cli/search.go

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) 2018, Sylabs Inc. All rights reserved.
+// This software is licensed under a 3-clause BSD license. Please consult the
+// LICENSE file distributed with the sources of this project regarding your
```

Comment:
LICENSE.md

or JMS gets mad

Reply:
Unless it's named LICENSE in the source. But it's not. :stuck_out_tongue: